src/sentry/features/manager.py

```diff
@@ -142,7 +142,7 @@ def all(self, feature_type: Type[Feature] = Feature) -> Mapping[str, Type[Featur
         Get a mapping of feature name -> feature class, optionally specific to a
         particular feature type.
         """
-        return {k: v for k, v in self._feature_registry.items() if v == feature_type}
+        return {k: v for k, v in self._feature_registry.items() if issubclass(v, feature_type)}
```

Comment:
Why is subclass more correct here?

Reply:
Because `v` is usually `OrganizationFeature` and the default value of `feature_type` was `Feature`. Meaning if you didn't pass anything to `all` it just wouldn't work.